testnet.template

```diff
@@ -25,9 +25,9 @@ if [ -z "$bioscontractpath" ]; then
     bioscontractpath="contracts/contracts/$bioscontractname"
 fi
 
-bioscurrencysymbol=$BIOS_CURRENCY_SYMBOL
+bioscurrencysymbol=$CORE_SYMBOL
 if [ -z "$bioscurrencysymbol" ]; then
-    bioscurrencysymbol="SYS"
+    bioscurrencysymbol="REM"
```

Comment:
we need to make sure tests pass if Protocol is built with SYS symbol

Reply:
@roman-tik, this bug comes from eos. When build eos by command `./scripts/eosio_build.sh -y -s EOS` and run tests `ctest -L nonparallelizable_tests --output-on-failure -T Test` parth of tests will fall.